local-setup/jupyter/dataloader/kubectl_api.py

```
@@ -0,0 +1,245 @@
+    data = request.json or {}
+    tenant_id = data.get('tenant_id')
+    env = data.get('env', 'chakshu')
+
+    if not tenant_id:
+        return jsonify({'error': 'tenant_id required'}), 400
+
+    try:
+        sql = f"""
+            DELETE FROM boundary_relationship WHERE tenantid='{tenant_id}';
+            DELETE FROM boundary WHERE tenantid='{tenant_id}';
+        """
```

Ruff (0.15.0)

[error] Possible SQL injection vector through string-based query construction

(S608)